### ISSUE TITLE
Add support for different background and font colors in sessions in Teleport Connect

### ISSUE THREAD

**Issue by philip-teleport** [gravitational/teleport#36754]
What would you like Teleport to do?

Add support to customize the background and font colors for different sessions/servers in Teleport Connect.

What problem does this solve?

In other terminal and SSH client tools, users have the flexibility to change colors and expect the same in Teleport Connect.

If a workaround exists, please include it.

N/A

**Comment by ravicious** [comment]
I assume you mean something like switching the background color to red when accessing production etc?

* https://bsago.me/tech-notes/change-ssh-background-colour-with-iterm2
  * https://iterm2.com/documentation-automatic-profile-switching.html
* https://superuser.com/questions/347512/can-i-have-my-terminal-background-change-based-on-hostname

Based on the last link, I wonder if it can be done completely in the shell without any external integration.

Here's [an issue in VSCode about this](https://github.com/microsoft/vscode-remote-release/issues/6567) which uses the same terminal emulator as we do. It was closed due to inactivity, but someone has linked to vscode-peacock which maybe solves this in some way.

**Comment by philip-teleport** [comment]
>  I assume you mean something like switching the background color to red when accessing production etc?

Yes, that's it. User would like different colours for different sessions so that it's easy to switch context.

**Comment by zmb3** [comment]
See also #26156 which is a more generic request to be able to differentiate the target cluster (whether it be via colors, fonts, or some other mechanism)

**Comment by zmb3** [comment]
For now we've solved this in a different way - by allowing users to associate a different color for each profile in the profile switcher.

See #51419 for a demo.
